docs/controls/BubbleSingleDataset.md

```diff
@@ -0,0 +1,327 @@
+# Bubble
```

Comment:
Can we change name to Bubble Single Dataset

Reply:
Updated here:
https://github.com/cerner/carbon-graphs/pull/308/commits/d5ccf1c09fb89321e361046b864e07099d3df278